python_modules/dagster/dagster/check/__init__.py

```diff
@@ -1274,9 +1516,13 @@ def _type_mismatch_error(
     )
```

Comment:
^ in `_type_mismatch_error` do we want to keep this "Desc: " prefix? I find it a bit weird. Maybe `Message: ` or something? Maybe just no prefix? [1] 

also worth desc -> additional_message in that fn impl i think

Reply:
good idea